.github/workflows/main.yaml

```diff
@@ -142,6 +142,20 @@ jobs:
         operating-system: [ "ubuntu-latest" ]
       fail-fast: false
 
+    services:
```

Comment:
Tohle neni potreba nebo jo?

Reply:
je, protože testy jsou závislé na db viz https://github.com/petrparolek/contributte-apitte-skeleton/actions jak padaly testy

Reply:
To mi nejak nejde do hlavy, jakto ze tedka testy prochazeji na aktualni master?

https://github.com/contributte/apitte-skeleton/runs/4294036683?check_suite_focus=true

Reply:
mě taky to nejde do hlavy, ale tak to je

Reply:
zřejmě jaká aktualizace composer balíčku

Reply:
Dropni to tu prosim. Chci videt ten error.

Reply:
:smile:  https://github.com/petrparolek/contributte-apitte-skeleton/runs/4313910920?check_suite_focus=true nejede

Reply:
To dá rozum. Jak chceš testovat aplikaci bez db?

Reply:
už to mám, ted ještě opravit generování configu v dockeru